README.md

```diff
@@ -148,6 +148,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Introduction to Data Science with R](https://www.youtube.com/watch?v=32o0DnuRjfg&list=PLTJTBoU5HOCRrTs3cJK-PbHM39cwCU0PF&index=1)
 - [Data Science Methodology](https://cognitiveclass.ai/courses/data-science-methodology-2)
 - [Data Science Cheat Sheets](https://www.kaggle.com/timoboz/data-science-cheat-sheets)
+- [Towards Data Science](https://towardsdatascience.com)
```

Comment:
As this is a blog rather than a link to a guide or a series, I don't think it qualifies. Please remove this one.

Reply:
Thank you for the observation. I just removed the reference.